### ISSUE TITLE
How can we use themes in our application

### ISSUE THREAD

**Issue by ilyamore88** [codex-team/task-manager#33]
Need research how we can use themes in our application. With this feature we can easily implement dark and other themes.
https://styled-components.com/docs/advanced#theming

**Comment by iegik** [comment]
For dark theme we can use media queries
```
  @media (prefers-color-scheme: dark) {
    --background-color: #1e1e1e;
    --color: #ffffffde;
    --color-overlay: #333333f0;
    --color-primary-contrast: #111111;
    --color-primary-dark: #54b3b8d9;
  }
```

**Comment by ilyamore88** [comment]
Hi, @iegik. We're using styled-component library in our project. There is an API for using themes by JS code and ThemeProvider. I've added the link on docs and you can check it.

I marked this issue as spike cause there is no design for themes. Need to check docs, create a ticket for designer with requirements, and create a ticket for developers after that.

But I think it isn't important for now.

**Comment by iegik** [comment]
@ilyamore88 Yeah, I sew the Styled documentation. But I think, that dark/light themes and "other" themes is a little bit different.
For example, `@media (prefers-color-scheme: dark)` checks if you have chosen "Dark" theme in OS and you can switch between light/dark. But with `ThemeProvider` you can force to choose concrete theme.

- Themes are good for personalising user experience on his authorised dashboard - when application recognises him.
- Dark mode support - is for MacOS users 

<img width="613" alt="Screenshot 2022-02-01 at 02 11 23" src="https://user-images.githubusercontent.com/175293/151888319-f8f830e6-9c82-4029-b27e-f16f5844273a.png">
when "Auto" option selected.

> But I think it isn't important for now.
P.S. Just getting to know the product and sharing my experience.
